app/models/need.rb

```diff
@@ -120,4 +121,14 @@ def assigned
   def last_phoned_date
     read_attribute('last_phoned_date')
   end
+
+  # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+  def self.sort_created_and_start_date(first, second)
```

Comment:
Maybe add a comment summarising what this does? e.g. "Sorts by start_on, then created_at" or something